### PR TITLE
release: v1.0.1

### PR DESCRIPTION
**Pull request by Xunzhuo**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

release: v1.0.1

### CODE REVIEW

File: release-notes/v1.0.1.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+date: April 9, 2024
+
+changes:
+  - area: installation
+    change: |
+      Bumped to Envoy v1.29.3
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:6]:
```suggestion
      Updated EnvoyProxy version to v1.29.3
```

Comment (Xunzhuo) [reply release-notes/v1.0.1.yaml]: Done

File: release-notes/v1.0.1.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+date: April 9, 2024
+
+changes:
+  - area: installation
+    change: |
+      Updated EnvoyProxy version to v1.29.3
+      Fixed Run certgen when upgrading
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:7]:
```suggestion
      Fixed certgen to support creating the hmac secret during an upgrade
```

File: release-notes/v1.0.1.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      Fixed set path prefix for http ext auth service
+      Fixed Change route sorting order to Exact > RegularExpression > PathPrefix
+      Fixed infraIR duplicate port translation for merged gateways
+      Fixed set SpawnUpstreamSpan to true
+      Fixed rate limit doesn't work with multiple listeners
+
+  - area: infra-manager
+    change: |
+      Fixed do not create infra resources when missing translated listeners
+      Fixed use Patch API for infra-client
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:26]: we never brought the Patch fix in, lets rm it

File: release-notes/v1.0.1.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      Fixed allow Policy to attach to multiple http listeners
+      Fixed set path prefix for http ext auth service
+      Fixed Change route sorting order to Exact > RegularExpression > PathPrefix
+      Fixed infraIR duplicate port translation for merged gateways
+      Fixed set SpawnUpstreamSpan to true
+      Fixed rate limit doesn't work with multiple listeners
+
+  - area: infra-manager
+    change: |
+      Fixed do not create infra resources when missing translated listeners
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:25]:
```suggestion
      Skip creating infra resources when the InfraIR has empty listeners
```

File: release-notes/v1.0.1.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+      Fixed add missing http filters to the http filter chain
+      Fixed allow websockets in url rewrite
+      Fixed set host for http health checker explicitly
+      Fixed double slashes in redirect URL
+      Fixed allow Policy to attach to multiple http listeners
+      Fixed set path prefix for http ext auth service
+      Fixed Change route sorting order to Exact > RegularExpression > PathPrefix
+      Fixed infraIR duplicate port translation for merged gateways
+      Fixed set SpawnUpstreamSpan to true
+      Fixed rate limit doesn't work with multiple listeners
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:21]:
```suggestion
      Allow rate limit to work with multiple listeners
```

File: release-notes/v1.0.1.yaml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+      Fixed nil secret in resourceversiontable
+      Fixed add missing http filters to the http filter chain
+      Fixed allow websockets in url rewrite
+      Fixed set host for http health checker explicitly
+      Fixed double slashes in redirect URL
+      Fixed allow Policy to attach to multiple http listeners
+      Fixed set path prefix for http ext auth service
+      Fixed Change route sorting order to Exact > RegularExpression > PathPrefix
+      Fixed infraIR duplicate port translation for merged gateways
+      Fixed set SpawnUpstreamSpan to true
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:20]:
```suggestion
      Set SpawnUpstreamSpan to true
```

File: release-notes/v1.0.1.yaml [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  - area: translator
+    change: |
+      Fixed nil secret in resourceversiontable
+      Fixed add missing http filters to the http filter chain
+      Fixed allow websockets in url rewrite
+      Fixed set host for http health checker explicitly
+      Fixed double slashes in redirect URL
+      Fixed allow Policy to attach to multiple http listeners
+      Fixed set path prefix for http ext auth service
+      Fixed Change route sorting order to Exact > RegularExpression > PathPrefix
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:18]:
```suggestion
      Set the route matching precedence order to Exact > RegularExpression > PathPrefix
```

File: release-notes/v1.0.1.yaml [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+      Fixed certgen to support creating the hmac secret during an upgrade
+
+  - area: translator
+    change: |
+      Fixed nil secret in resourceversiontable
+      Fixed add missing http filters to the http filter chain
+      Fixed allow websockets in url rewrite
+      Fixed set host for http health checker explicitly
+      Fixed double slashes in redirect URL
+      Fixed allow Policy to attach to multiple http listeners
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:16]:
```suggestion
      Allow ClientTrafficPolicy to attach to multiple http (non https) listeners within the same Gateway
```

File: release-notes/v1.0.1.yaml [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    change: |
+      Updated EnvoyProxy version to v1.29.3
+      Fixed certgen to support creating the hmac secret during an upgrade
+
+  - area: translator
+    change: |
+      Fixed nil secret in resourceversiontable
+      Fixed add missing http filters to the http filter chain
+      Fixed allow websockets in url rewrite
+      Fixed set host for http health checker explicitly
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:14]:
```suggestion
      Set the Host header for http health checker
```

File: release-notes/v1.0.1.yaml [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+  - area: installation
+    change: |
+      Updated EnvoyProxy version to v1.29.3
+      Fixed certgen to support creating the hmac secret during an upgrade
+
+  - area: translator
+    change: |
+      Fixed nil secret in resourceversiontable
+      Fixed add missing http filters to the http filter chain
+      Fixed allow websockets in url rewrite
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:13]:
```suggestion
      Allow websockets when url rewrite is enabled
```

File: release-notes/v1.0.1.yaml [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+changes:
+  - area: installation
+    change: |
+      Updated EnvoyProxy version to v1.29.3
+      Fixed certgen to support creating the hmac secret during an upgrade
+
+  - area: translator
+    change: |
+      Fixed nil secret in resourceversiontable
+      Fixed add missing http filters to the http filter chain
```
Comment (arkodg) [inline release-notes/v1.0.1.yaml:12]:
```suggestion
      Add missing http filters to the http filter chain
```

**Comment by arkodg** [comment]
> can we just reuse doc from v1.0.0?

@zirain can you elaborate ?

**Comment by zirain** [comment]
> > can we just reuse doc from v1.0.0?
> 
> @zirain can you elaborate ?

there's little change around doc between v1.0.0 and v1.0.1, so should not create a new version of doc?

**Comment by zirain** [comment]
> > > > can we just reuse doc from v1.0.0?
> > > 
> > > 
> > > @zirain can you elaborate ?
> > 
> > 
> > there's little change around doc between v1.0.0 and v1.0.1, so should not create a new version of doc?
> 
> We either need to set docs to v1.0.1 or rename dir to v1.0, I believe former is easier cc @Xunzhuo

At least, should do this later

**Comment by Xunzhuo** [comment]
> can we just reuse doc from v1.0.0?

Actually, it is reused from v1.0.0, just changed path and modified the related image tags in quickstart. We do not generate new paths for patch release, if we do so, there will be two paths for v1.0.0 and v1.0.1.

File: .github/workflows/docs.yaml [hunk 12]
```diff
@@ -40,6 +40,7 @@ jobs:
 
     - name: Check links
       run: make docs docs-check-links
+      continue-on-error: true
```
Comment (zirain) [inline .github/workflows/docs.yaml:43]: wait https://github.com/envoyproxy/gateway/pull/3146

File: site/content/en/v1.0.1/user/quickstart.md [hunk 13]
```diff
@@ -43,7 +43,7 @@ unprivileged port, so that Envoy Gateway doesn't need additional privileges.
 It's important to be aware of this mapping, since you may need to take it into
 consideration when debugging.
 
-[`quickstart.yaml`]: https://github.com/envoyproxy/gateway/releases/download/v1.0.0/quickstart.yaml
+[`quickstart.yaml`]: https://github.com/envoyproxy/gateway/releases/download/v1.0.1/quickstart.yaml
```
Comment (Xunzhuo) [inline site/content/en/v1.0.1/user/quickstart.md:46]: Some links are 404 as expected since release is not cut off yet

**Review by arkodg — APPROVED** [review]

File: .github/workflows/docs.yaml (hunk 12 above)
Comment (zirain) [reply .github/workflows/docs.yaml:43]: I think you can remove this now.

**Review by zirain — APPROVED** [review]

File: site/content/en/v1.0.1/user/quickstart.md (hunk 13 above)
Comment (zirain) [reply site/content/en/v1.0.1/user/quickstart.md:46]: this link will never 404, because github.com is ignored because of rate limiting.

**Pull request merged**